src/main/cpp/subsystems/LimeLightUtil.cpp

```diff
@@ -0,0 +1,50 @@
+#include "subsystems/LimeLightUtil.h"
+
+t34::LimelightUtil::LimelightUtil(TrajMath math_handler, TargetMode target_mode)
+:   m_limelight_swerve_pid(0.1, 0.0, 0.1),
+    m_table(LimelightHelpers::getLimelightNTTable(LIMELIGHT_TABLE_NAME)),
+    m_tx(LimelightHelpers::getTX(LIMELIGHT_TABLE_NAME)),
+    m_ty(LimelightHelpers::getTY(LIMELIGHT_TABLE_NAME)),
+    m_heading_error(-m_tx),
+    m_kp(-0.1),
+    m_min_command(0.5),
+    m_steering_adjust(0.0),
+    m_drive_x(0.0),
+    m_drive_y(0.0),
+    m_swerve_drive_speeds{0.0, 0.0, 0.0},
+    m_math_handler(math_handler),
+    m_target_mode(target_mode) 
+{}
+
+void t34::LimelightUtil::AdjustSteering()
+{
+    if (fabs(m_heading_error) > 1.0) {
+
+        if (m_heading_error < 0.0) {
+
+            m_steering_adjust = m_kp*m_heading_error + m_min_command;
+
+        }
+        else 
+        {
+
+            m_steering_adjust = m_kp*m_heading_error - m_min_command;
+      
+    }
```

Comment:
'Tis a minor thing, but are you familiar with the ternary operator? 

`condition ? value_if_true : value_if_false`

When I see both the `if` and `else` branches setting the same variable, I usually reach for it:

```suggestion
        m_steering_adjust = m_heading_error < 0.0 ?
            m_kp * m_heading_error + m_min_command :
            m_kp * m_heading_error - m_min_command;
```

… or even …

```suggestion
        const auto adjustment = m_heading_error < 0.0 ? m_min_command : -m_min_command;
        m_steering_adjust = m_kp * m_heading_error + adjustment;
```

If either of the above (or some variant thereof) appeals to you, fantastic. If not, feel free to reject.